metrics/sinks/stackdriver/stackdriver.go

```diff
@@ -175,13 +175,18 @@ func (sink *StackdriverSink) ExportData(dataBatch *core.DataBatch) {
 	sink.lastExportTime = dataBatch.Timestamp
 
 	req := getReq()
-	for _, metricSet := range dataBatch.MetricSets {
+	for key, metricSet := range dataBatch.MetricSets {
 		switch metricSet.Labels["type"] {
 		case core.MetricSetTypeNode, core.MetricSetTypePod, core.MetricSetTypePodContainer, core.MetricSetTypeSystemContainer:
 		default:
 			continue
 		}
 
+		if metricSet.CreateTime.IsZero() {
+			glog.Infof("Skipping incorrect metric set %s because create time is zero", key)
```

Comment:
If someone sees this in logs, what this person does? If the answer is "nothing", I think this definitely this like should not be V(0)

Ideally, I'd prefer dropping this to V(2) at least and record a metric if it's not possible to figure it out from existing metrics already. This way the losses are visible, but the log is not flooded